### PR TITLE
Add FAQ text

### LINKED ISSUE #1366: Add text to FAQ page
Sent via email. Perhaps a developer adds this and then design reviews if necessary.

### PR DESCRIPTION
**Pull request by RickMohr**
Fixes #1366

### CODE REVIEW

**Comment by RickMohr** [comment]
@jfrankl or @designmatty, want to style this?

**Comment by RickMohr** [comment]
I talked with @jfrankl and added basic styling.

File: src/nyc_trees/apps/home/templates/home/about_faq.html [hunk 1]
```diff
@@ -10,6 +10,95 @@
 
 {% block main %}
 
-Additional details coming soon. Learn more at <a href="http://nyc.gov/parks/treescount">NYC Parks!</a>
+<h4>What is TreesCount! 2015?</h4>
+<p>
+    TreesCount! 2015 is NYC Parks&rsquo; effort to map and catalogue every street
+    tree on every block in New York City.
+    <a href="{% url 'registration_register' %}">Register</a>
```
Comment (steventlamb) [inline src/nyc_trees/apps/home/templates/home/about_faq.html]: fwiw - I see this when I'm logged in.

**Comment by steventlamb** [comment]
+1

**Pull request merged**